benchmark/single_model_benchmark.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative 'performance_helper'
+
+ForkingBenchmark.ips do |x|
+  x.time = 5
```

Comment:
is there any reason for 5 and 2 or just because they sounded fine?

Reply:
They were copied from the benchmark that landed in #77. Looks like these are the defaults for benchmark-ips though so I'll remove them.